_data/government.yml

```diff
@@ -1,4 +1,13 @@
 websites:
+  - name: AFIP
+    url: https://www.afip.gob.ar
+    img: afip.png
+    tfa:
+      - hardware
+      - proprietary
+    doc: https://serviciosweb.afip.gob.ar/genericos/GuiaDeTramites/VerGuia.aspx?tr=38
+    exception: "2FA is required for users with Clave Fiscal (security level) 4"
```

Comment:
>is required

But hardware and software tokens can still be used for security levels <4?

Reply:
No, to request a hardware token or be able to use the software one you must be on level 4. Levels 1, 2, and 3 are not allowed to use 2FA.

Reply:
```suggestion
    exception: "2FA is only available for users with security level (Clave Fiscal) 4"
```